devnet-sdk/system2/l1_cl.go

```diff
@@ -0,0 +1,57 @@
+package system2
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+)
+
+type L1CLNodeID idWithChain[string]
+
+func (id L1CLNodeID) String() string {
+	return idWithChain[string](id).string("L1CLNode")
+}
+
+func (id L1CLNodeID) MarshalText() ([]byte, error) {
+	return idWithChain[string](id).marshalText("L1CLNode")
+}
+
+func (id *L1CLNodeID) UnmarshalText(data []byte) error {
+	return (*idWithChain[string])(id).unmarshalText("L1CLNode", data)
+}
+
+type L1CLNode interface {
+	Common
+	ID() L1CLNodeID
+
+	sources.BeaconClient
```

Comment:
The same as above - I would rather provide an accessor than wrapping this type

Reply:
Sure, that makes sense